### PR TITLE
tealdeer: update to 1.6.0

### PR DESCRIPTION
**Pull request by tranzystorek-io**
#### Testing the changes
- I tested the changes in this PR: **briefly**

### CODE REVIEW

**Comment by Piraty** [comment]
ping @jcgruenhage

**Comment by jcgruenhage** [comment]
lgtm :)

**Comment by tranzystorek-io** [comment]
@jcgruenhage if you don't like your maintained packages bumped by others, let me know, don't want to be annoying

**Comment by jcgruenhage** [comment]
no, don't worry, feel free to bump those packages. I like a ping to give approval on it because of experiences in the past where random people updated the package without reading the changelog, breaking my setup in the progress...

As long as people take the required care while updating packages I'm very happy about people bumping the packages, less work for me ;)

File: srcpkgs/tealdeer/template [hunk 1]
```diff
@@ -10,15 +10,15 @@ maintainer="jcgruenhage <jan.christian@gruenhage.xyz>"
 license="MIT, Apache-2.0"
 homepage="https://github.com/dbrgn/tealdeer"
 changelog="https://raw.githubusercontent.com/dbrgn/tealdeer/v${version}/CHANGELOG.md"
```
Comment (classabbyamp) [inline srcpkgs/tealdeer/template:12]:
```suggestion
changelog="https://raw.githubusercontent.com/dbrgn/tealdeer/main/CHANGELOG.md"
```

**Pull request merged**